pkg/webhook/resources/restore/validator.go

```diff
@@ -92,6 +96,12 @@ func (v *restoreValidator) Create(request *types.Request, newObj runtime.Object)
 		return werror.NewInvalidError(fmt.Sprintf("please stop the VM %q before doing a restore", vm.Name), fieldTargetName)
 	}
 
+	if result, err := util.HasInProgressingVMRestoreOnSameTarget(v.vmRestore, vm.Namespace, vm.Name); err != nil {
+		return werror.NewInternalError(fmt.Sprintf("can't check whether there is another in progressing vmrestore, err: %+v", err))
+	} else if result {
+		return werror.NewInvalidError(fmt.Sprintf("please wait for previous vmrestore complete on the same target %s/%s", vm.Namespace, vm.Name), fieldTargetName)
```

Comment:
The feature works well.
How about let's rephrase the messages more user-friendly:
<img width="973" alt="Screen Shot 2022-07-29 at 12 21 40 PM" src="https://user-images.githubusercontent.com/1691518/181682412-fe4eaa9f-9941-4aeb-bf2c-405d4b5a26cc.png">

`vmrestore` looks like an internal variable naming.

Reply:
ditto, may change the words to `please wait for the previous VM restore on the $ns/name to be complete first.`

Reply:
Fixed it in https://github.com/harvester/harvester/pull/2601. Thank you.